SpellCheck/pom.xml

```diff
@@ -25,43 +17,41 @@
 			<comments>Educational Online Test Delivery System Copyright (c) 2013 American Institutes for Research</comments>
 		</license>
 	</licenses>
+
 	<dependencies>
+		<dependency>
+			<groupId>com.atlascopco</groupId>
+			<artifactId>hunspell-bridj</artifactId>
+			<version>1.0.4</version>
+		</dependency>
```

Comment:
This hunspell java bridge implementation is available from Maven Central. (Available under Mozilla Public License 1.1)